client/src/templates/Introduction/components/block.tsx

```diff
@@ -203,9 +201,7 @@ export class Block extends Component<BlockProps> {
           >
             <Caret />
             <h4 className='course-title'>
-              {`${
-                isExpanded ? collapseText : expandText
-              } ${coursesText.toLowerCase()}`}
```

Comment:
Considering this, please make the word courses lowercase to avoid changing the existing UI for English

Reply:
Hi, I made changes as requested.